xenit-alfresco/templates/ingress/alfresco-ingress.yaml

```
@@ -44,27 +58,29 @@ spec:
         number: {{ .Values.ingress.defaultBackend.port }}
   {{- end }}
   rules:
-  - host: {{ required "A host where your alfresco services can be reached on must be specified in values.ingress.host" .Values.ingress.host }}
+  {{- $values := .Values }} # We will break the reference to .Values on the next line (until we escape the range).
+  {{- range $allAlfrescoHostNames }}
+  - host: {{ . | quote }}
     http:
       paths:
-      {{- if .Values.general.hibernate }}
+      {{- if $values.general.hibernate }}
```

```suggestion
      {{- if $.Values.general.hibernate }}
```

You can use `$` to refer to the root object when inside a different context

---

Implemented improvement.